src/main/java/io/strimzi/kafka/bridge/config/BridgeConfig.java

```diff
@@ -6,23 +6,39 @@
 package io.strimzi.kafka.bridge.config;
 
 import io.strimzi.kafka.bridge.http.HttpConfig;
+import io.strimzi.kafka.bridge.metrics.MetricsType;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
+import java.nio.file.Path;
 import java.util.Map;
+import java.util.Optional;
 import java.util.stream.Collectors;
 
 /**
  * Bridge configuration properties
  */
 public class BridgeConfig extends AbstractConfig {
+    private static final Logger LOGGER = LogManager.getLogger(BridgeConfig.class);
 
     /** Prefix for all the specific bridge configuration parameters */
     public static final String BRIDGE_CONFIG_PREFIX = "bridge.";
 
     /** Bridge identification number */
     public static final String BRIDGE_ID = BRIDGE_CONFIG_PREFIX + "id";
 
+    /** Metrics system to be used in the bridge */
+    public static final String METRICS_TYPE = BRIDGE_CONFIG_PREFIX + "metrics";
+
+    /** JMX Exporter configuration file path */
+    private static final String JMX_EXPORTER_CONFIG_PATH = METRICS_TYPE + ".exporter.config.path";
+    
     /** Tracing system to be used in the bridge */
-    public static final String TRACING_TYPE = BRIDGE_CONFIG_PREFIX + "tracing";
+    private static final String TRACING_TYPE = BRIDGE_CONFIG_PREFIX + "tracing";
+    
+    /** Default Strimzi Metrics Reporter allow list. */
+    /* test */ static final String DEFAULT_STRIMZI_METRICS_REPORTER_ALLOW_LIST = "kafka_consumer_consumer_metrics.*, " +
+        "kafka_producer_kafka_metrics_count_count, kafka_producer_producer_metrics.*";
```

Comment:
More out of curiosity than anything else ... what does `kafka_producer_kafka_metrics_count_count` show?

Reply:
It's Kafka producer instances, used by the operator's Bridge example dashboard.